site/mosaic.config.js

```diff
@@ -23,6 +23,26 @@ const saltConfig = {
       modulePath: "@jpmorganchase/mosaic-plugins/BreadcrumbsPlugin",
       disabled: true,
     },
+    {
+      modulePath: require.resolve(
+        "./src/mosaic-plugins/LabsComponentPlugin.mjs"
+      ),
+      options: {
+        componentsList: [
+          "Badge",
```

Comment:
Badge seems to be missing the Lab icon for some reason

Reply:
Should be fine now